src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MirrorEndpointsSpec.groovy

```diff
@@ -1183,6 +1198,47 @@ class MirrorEndpointsSpec extends HealthCheckSpecification {
         return northbound.getSwitchProperties(switchId)
     }
 
+    /**
+     * Identify all unique switches that need to be tested. Return minimum amount of switch pairs, where switch pair
+     * satisfies 'additional conditions' and src switch is a switch under test. Sources with 2 traffgens have a higher priority
+     */
+    @Memoized
+    List<SwitchPair> getUniqueSwitchPairs(Closure additionalConditions = { true }) {
```

Comment:
shouldn't we exclude the `WB` switch till the issue is fixed?

Reply:
Yes. Will now raise the github issue and exclude WB switches with link